gateway/helm/templates/deployment.yaml

```diff
@@ -89,6 +87,12 @@ spec:
             - {{- . | toYaml | trim | nindent 14 }}
             {{- end }}
             {{- end }}
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                  - sleep
+                  - 15
```

Comment:
Not a huge fan of sleeping to fix the underlying cause. Let's understand the reason for the timeouts before we apply this strategy.